setup.py

```diff
@@ -91,19 +91,46 @@ def build_extension(self, ext: CMakeExtension):
                     "-DCMAKE_SYSTEM_PROCESSOR=ARM64",
                 ]
             else:  # X64 arch
-                if shutil.which("brew"):
+                if os.getenv("USE_BREW_LLVM") and shutil.which("brew"):
+                    brew_llvm_version = os.getenv("BREW_LLVM_VERSION")
```

Comment:
If `BREW_LLVM_VERSION` is not a common variable, maybe we could simply check whether `BREW_LLVM_VERSION` is set and let go of `USE_BREW_LLVM`.

Reply:
Yep, that's a good idea, and much cleaner.

Reply:
Then I think we just need to replace `USE_BREW_LLVM` by `BREW_LLVM_VERSION`.
```suggestion
                if os.getenv("BREW_LLVM_VERSION") and shutil.which("brew"):
                    brew_llvm_version = os.getenv("BREW_LLVM_VERSION")
```